frontend/www/js/omegaup/components/arena/ContestPractice.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div data-contest-practice>
+    <audio v-if="admin" class="notification-audio">
```

Comment:
```suggestion
    <audio v-if="admin" data-notification-audio>
```

ahorita se está buscando por ID en Clarifications.vue, así que esto va a fallar de cualquier modo con una clase. entonces es mejor migrarlo a un data attribute de una vez.

de paso, este audio debería ir en Clarifications.vue, porque ahorita hay una desconexión entre donde se declara y donde se usa, y los componentes deberían de no tener ese problema.